### PR TITLE
Add some missing references

### PR DESCRIPTION
**Pull request by ycmin95**
Added missing references: cheng2020fully, min2021visual, jiao2023cosign, zhou2023gloss, jiao2024visual.
Plan to include more references from [VIPL-SLP/awesome-sign-language-processing](https://github.com/VIPL-SLP/awesome-sign-language-processing) in future updates.

### CODE REVIEW

**Review by AmitMY — CHANGES_REQUESTED** [review]
Thank you very much for your contribution, happy to see you join the effort :) 

I left a few comments for minor changes.

For future PRs, I find it easier to review PRs that add a single paragraph / refer to a single topic (with a few citations). It makes it so I can review even if I have less time, and merge some changes while waiting for changes for others (just a preference).

**Comment by ycmin95** [comment]
Thank you for your comprehensive feedback. I have revised relevant parts and restructured the sentences for clarity. Additionally, I’ve discovered that incorporating hyperlinks for each reference can greatly enhance the document’s usability.

File: src/references.bib [hunk 1]
```diff
@@ -1546,6 +1546,14 @@ @article{jiang2021sign
  year = {2021}
 }
 
+@inproceedings{jiao2023cosign,
+  title={CoSign: Exploring co-occurrence signals in skeleton-based continuous sign language recognition},
```
[the remark this answers is not in the document]
Comment (ycmin95) [reply src/references.bib]: Thanks for the suggestion, I revised this reference item.

File: src/index.md [hunk 2]
```diff
@@ -538,10 +538,9 @@ Though some previous works have referred to this as "sign language translation,"
...
...(SAM-SLR-v2) to learn and fuse multimodal feature representations. Specifically, they use a Sign Language Graph Convolution Network (SL-GCN) to model the embedded dynamics of skeleton keypoints and a Separable Spatial-Temporal Convolution Network (SSTCN) to exploit skeleton features. The proposed late-fusion GEM fuses the skeleton-based predictions with other RGB and depth-based modalities to provide global information and make an accurate SLR prediction. @jiao2023cosign explore co-occurence signals in skeleton data to better exploit the knowledge of each signal for continuous SLR.  Specifically, they use Group-specific GCN to abstract skeleton features from co-occurence signals (Body, Hand, Mouth and Hand) and introduce complementary regularization to ensure consistency between predictions based on two complementary subsets of signals. Additionally, they propose a two-stream framework to fuse static and dynamic information. The model demonstrates competitive performance cpmpared to video-to-gloss methods on the RWTH-PHOENIX-Weather-2014 [@koller2015ContinuousSLR], RWTH-PHOENIX-Weather-2014T [@cihan2018neural] and CSL-Daily [@dataset:Zhou2021_SignBackTranslation_CSLDaily] datasets.
```
[the remark this answers is not in the document]
Comment (ycmin95) [reply src/index.md]: I have divided this paragraph into individual sentences to more clearly highlight the distinctions. Besides, in the previous version, I changed the tense of the previous sentence from past tense to present tense (@jiang2021sign proposed -> @jiang2021sign propose), and I recover the original version in the updated version. 

Currently, it appears that the tenses in this project are not consistent and may require an overall review and correction.

File: src/index.md [hunk 3]
```diff
@@ -538,10 +538,9 @@ Though some previous works have referred to this as "sign language translation,"
...
 
-@dafnis2022bidirectional work on the same modified WLASL dataset as @jiang2021sign, but do not require multimodal data input. Instead, they propose a bidirectional skeleton-based graph convolutional network framework with linguistically motivated parameters and attention to the start and end
-frames of signs. They cooperatively use forward and backward data streams, including various sub-streams, as input. They also use pre-training to leverage transfer learning.
+@dafnis2022bidirectional work on the same modified WLASL dataset as @jiang2021sign, but do not require multimodal data input. Instead, they propose a bidirectional skeleton-based graph convolutional network framework with linguistically motivated parameters and attention to the start and end frames of signs. They cooperatively use forward and backward data streams, including various sub-streams, as input. They also use pre-training to leverage transfer learning.
```
[the remark this answers is not in the document]
Comment (ycmin95) [reply src/index.md:546]: Thanks for pointing out, I am not accustomed to the boundary here, so I have made changes here :）

File: src/index.md [hunk 4]
```diff
@@ -587,7 +586,11 @@ For this recognition, @cui2017recurrent constructs a three-step optimization mod
 First, they train a video-to-gloss end-to-end model, where they encode the video using a spatio-temporal CNN encoder
 and predict the gloss using a Connectionist Temporal Classification (CTC) [@graves2006connectionist].
 Then, from the CTC alignment and category proposal, they encode each gloss-level segment independently, trained to predict the gloss category,
-and use this gloss video segments encoding to optimize the sequence learning model.
+and use this gloss video segments encoding to optimize the sequence learning model. @cheng2020fully propose a fully convolutional networks for continuous SLR,
```
[the remark this answers is not in the document]
Comment (ycmin95) [reply src/index.md]: Thanks for the suggestion, I revised this sentence.

File: src/index.md [hunk 5]
```diff
@@ -587,7 +586,11 @@ For this recognition, @cui2017recurrent constructs a three-step optimization mod
 First, they train a video-to-gloss end-to-end model, where they encode the video using a spatio-temporal CNN encoder
 and predict the gloss using a Connectionist Temporal Classification (CTC) [@graves2006connectionist].
 Then, from the CTC alignment and category proposal, they encode each gloss-level segment independently, trained to predict the gloss category,
-and use this gloss video segments encoding to optimize the sequence learning model.
+and use this gloss video segments encoding to optimize the sequence learning model. @cheng2020fully propose a fully convolutional networks for continuous SLR, 
+moving away from LSTM-based methods to achieve end-to-end learning. They introduce a gloss feature enhancement (GFE) module to provide additional rectified supervision and
```
[the remark this answers is not in the document]
Comment (ycmin95) [reply src/index.md]: Thanks for the suggestion, I revised this sentence.

File: src/index.md [hunk 6]
```diff
@@ -792,6 +798,10 @@ and showed similar performance, with the transformer underperforming on the vali
 They experimented with various normalization schemes, mainly subtracting the mean and dividing by the standard deviation of every individual keypoint
 either concerning the entire frame or the relevant "object" (Body, Face, and Hand).
 
+@jiao2024visual propose a visual alignment pre-training framework for gloss-free sign language translation. Specifically, they adopt Cosign-1s [@jiao2023cosign] to obtain skeleton features from estimated pose sequences
```
[the remark this answers is not in the document]
Comment (ycmin95) [reply src/index.md]: CoSign, thanks!

File: src/index.md [hunk 7]
```diff
@@ -587,7 +586,11 @@ For this recognition, @cui2017recurrent constructs a three-step optimization mod
 First, they train a video-to-gloss end-to-end model, where they encode the video using a spatio-temporal CNN encoder
 and predict the gloss using a Connectionist Temporal Classification (CTC) [@graves2006connectionist].
 Then, from the CTC alignment and category proposal, they encode each gloss-level segment independently, trained to predict the gloss category,
-and use this gloss video segments encoding to optimize the sequence learning model.
+and use this gloss video segments encoding to optimize the sequence learning model. @cheng2020fully propose a fully convolutional networks for continuous SLR, 
+moving away from LSTM-based methods to achieve end-to-end learning. They introduce a gloss feature enhancement (GFE) module to provide additional rectified supervision and 
+accelerate the training process. @min2021visual attribute the success of iterative training to its ability to reduce overfitting. They propose visual enhancement
+constraint (VEC) and visual alignment constraint (VAC) to strengthen the visual extractor and align long- and short-term predictions, enabling LSTM-based methods to be trained in an end-to-end manner.
```
[the remark this answers is not in the document]
Comment (ycmin95) [reply src/index.md]: Thanks for the suggestion, I have capitalized them.

File: src/index.md [hunk 8]
```diff
@@ -742,6 +745,10 @@ The model features shared representations for different modalities such as text
 on several tasks such as video-to-gloss, gloss-to-text, and video-to-text. 
 The approach allows leveraging external data such as parallel data for spoken language machine translation.
 
+@zhou2023gloss propose the GFSLT-VLP framework for gloss-free sign language translation, which improves SLT performance through visual-alignment pretraining. In the pretraining stage, they design a pretext task that aligns visual and textual
```
[the remark this answers is not in the document]
Comment (ycmin95) [reply src/index.md]: Thanks for the suggestion, I have revised this sentence.

File: src/index.md [hunk 9]
```diff
@@ -792,6 +798,10 @@ and showed similar performance, with the transformer underperforming on the vali
 They experimented with various normalization schemes, mainly subtracting the mean and dividing by the standard deviation of every individual keypoint
 either concerning the entire frame or the relevant "object" (Body, Face, and Hand).
 
+@jiao2024visual propose a visual alignment pre-training framework for gloss-free sign language translation. Specifically, they adopt Cosign-1s [@jiao2023cosign] to obtain skeleton features from estimated pose sequences
+and a pretrained text encoder to obtain corresponding textual features. During pretraining, these visual and textual features are aligned in a greedy manner. In the finetuning stage, they replace the shallow translation module 
+used in pretraining with a pretrained translation module. This skeleton-based approach achieves state-of-the-art results on the RWTH-PHOENIX-Weather-2014T [@cihan2018neural], CSL-Daily [@dataset:Zhou2021_SignBackTranslation_CSLDaily], OpenASL [@shi-etal-2022-open], and How2Sign[@dataset:duarte2020how2sign] datasets without relying on gloss annotations.
```
[the remark this answers is not in the document]
Comment (ycmin95) [reply src/index.md]: The space has been added.

**Review by AmitMY — APPROVED** [review]
Great job!

**Pull request merged**